app/main.py

```diff
@@ -16,4 +16,58 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+"""
+# parse the names of first n variables in a file
+def get_n_variable_names(file, n):
+    try:
+        with open(file, "r") as f:
+            variables = []
+            while len(variables) != n:
+                line = f.readline().split(" = ")[0]
+                if line[0].isalpha() and "import" not in line:
+                    variables.append(line)
+            return variables
+    except FileNotFoundError:
+        print(f"{file} not found")
+        return None
+
+variable_names = get_n_variable_names("main.py", 8)
```

Comment:
The function 'get_n_variable_names' reads from a file named 'main.py', but the file path might need to be adjusted depending on the actual file location. Ensure the correct path is used when calling this function.